Sources/MongoSwift/EventLoopBoundMongoClient.swift

```diff
@@ -0,0 +1,98 @@
+import NIO
+
+// sourcery: skipSyncExport
+/// An event loop bound MongoDB Client.
```

Comment:
I think we should rewrite this docstring to use different words than are in the type itself to explain what being bound means, similar to the docstring on the `bound` method.